crates/relayer/src/chain/cosmos/compatibility.rs

```diff
@@ -20,7 +20,7 @@ const SDK_MODULE_VERSION_REQ: &str = ">=0.44, <0.48";
 /// # Note: Should be consistent with [features] guide page.
 ///
 /// [features]: https://hermes.informal.systems/advanced/features.html
-const IBC_GO_MODULE_VERSION_REQ: &str = ">=1.1, <=7";
+const IBC_GO_MODULE_VERSION_REQ: &str = ">=1.1, <9";
```

Comment:
The 1.1 minimum version seems low, should we increase it to v4 at least?
I don't recall seeing any chains using lower than v4 and in theory ibc-go v6.1 and v6.2 reached end of life the 9th of December 2023 (but there are still many chains on v4)

Reply:
Good point, yeah let's raise it to v4 for now and v7 in the next Hermes release.